rules/windows/initial_access_evasion_suspicious_htm_file_creation.toml

```diff
@@ -17,6 +17,7 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious HTML File Creation"
+note = "This rule may have a low to medium performance impact due variety of file paths potentially matching each EQL sequence."
```

Comment:
wouldn't be a performance tag easier for users to be able to filter out via GUI from rules management menu ?

Reply:
Yes, performance tags would be a good filter data point, however, since we have no current method to properly measure performance, the tags would be subjective and predicated on environment. Unless we have specific notes that would allow us to differentiate, such as maxspan > 1m -> medium performance. If we have these rules, then we could programmatically address all rules and add unit testing.